File: CoderBytes/py/exam-fizzbuzz.py
def fizzbuzz(n: int):
    """
    the **fizzbuzz** game\
    (int n)
    @param int n the maximum number  
    """
    l = []
    for i in [ 'FizzBuzz' if (x % 3 == 0) and (x % 5 == 0) else 'Fizz' if x % 3 == 0 else 'Buzz' if x % 5 == 0 else x for x in range(1, n + 1)]:
        l.append(str(i))
    
    return ' '.join(l)

print(fizzbuzz(16))

File: CoderBytes/php/exam-fizzbuzz.php
<?php
function fizzbuzz(int $num){
    $t = [];
    for($i = 1; $i<=$num; $i++){
        $ch = '';
        $ch .= (($i%5)==0) ? 'fizz':'';
        $ch .= (($i%3)==0) ? 'buzz':'';
        $t[] = !empty($ch)? $ch : $i;
    }

    return implode(' ', $t);
}

echo fizzbuzz(16)."\n";
